cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -30,12 +30,22 @@ <h2>Application requirements</h2>
     <dl class="m-list m-list__card-details">
         <dt>Location</dt>
         <dd>
-            {% if not card.state_limitations %}
+            {%- if card.availability_of_credit_card_plan == "One State/Territory" %}
+                {% set state_limitations = [card.state] %}
+            {% elif card.availability_of_credit_card_plan == "Regional" %}
+                {% set state_limitations = card.state_multiple %}
+            {% else %}
+                {% set state_limitations = none %}
+            {% endif -%}
+
+            {% if not state_limitations %}
                 Available to anyone in the United States
             {% else %}
                 Available only to people who live in
                 {{ 'specific counties in ' if card.pertains_to_specific_counties }}
-                {{ card.state_limitations | join(", ") }}
+                {% for state in state_limitations %}
+                    {{ state_lookup[state] }}{% if not loop.last %},{% endif %}
```

Comment:
I think to cleanly handle cases with just two states and to get our Oxford comma in, this should work:

```suggestion
                    {{ state_lookup[state] -}}
                    {{- ', ' if not loop.last and loop.length > 2 }}
                    {{ 'and ' if loop.index == loop.length - 1}}
```

Reply:
Thanks @niqjohnson, that's a great suggestion. I've committed your change with some additional whitespace collapsing in 3ab7617.